stl/inc/deque

```diff
@@ -1673,6 +1682,16 @@ private:
         return _Get_data()._Mysize;
     }
 
+    _Map_difference_type _Map_distance() const noexcept {
+        return static_cast<_Map_difference_type>(_Get_data()._Mapsize);
+    }
+
+    _Ty* _Address_at(size_type _Idx) noexcept {
+        const auto _Block = _Getblock(_Idx);
+        const auto _Off   = static_cast<difference_type>(_Idx % _Block_size);

```

Comment:
this should belong to `_Deque_val`, as it's taking `_Idx` as map-index instead of public-index.